internal/manifests/collector/service.go

```diff
@@ -108,6 +110,40 @@ func MonitoringService(params manifests.Params) (*corev1.Service, error) {
 	}, nil
 }
 
+func ExtensionService(params manifests.Params) (*corev1.Service, error) {
+	name := naming.Service(params.OtelCol.Name)
+	labels := manifestutils.Labels(params.OtelCol.ObjectMeta, name, params.OtelCol.Spec.Image, ComponentOpenTelemetryCollector, []string{})
+	labels[extensionService] = valueExists
```

Comment:
I don't think you actually need this. We are using this label to create `ServiceMonitors`.